sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineRunner.java

```diff
@@ -152,6 +165,12 @@
   ////////////////////////////////////////////////////////////////////////////////////////////////
   private final InProcessPipelineOptions options;
 
+  public static InProcessPipelineRunner createForTest() {
+    InProcessPipelineOptions options = PipelineOptionsFactory.as(InProcessPipelineOptions.class);
+    options.setBlockOnRun(true);
```

Comment:
This is temporary for exact parity until we settle on how to write streaming unit tests?